content/rest/channels-messages.textile

```diff
@@ -336,13 +361,11 @@ Gets a "paginated":#paginated-result set of historical messages for this channel
 
 h4. Parameters
 
-- <span lang="default">options</span><span lang="objc,swift">query</span><span lang="java">"Param":#param[] options</span> := <span lang="default">an optional object containing the query parameters</span><span lang="ruby">an optional set of key value pairs containing the query parameters</span><span lang="ruby">an optional Associative Array containing the query parameters</span>, as specified in the "message history API documentation":/realtime/history#channel-history.
+- <span lang="default">options</span><span lang="objc,swift">query</span><span lang="java">"Param":#param[] options</span> := <span lang="default">an optional object containing the query parameters</span><span lang="python">optional keyword arguments containing the query parameters</span><span lang="ruby">an optional set of key value pairs containing the query parameters</span><span lang="ruby">an optional Associative Array containing the query parameters</span>, as specified in the "message history API documentation":/rest/history#channel-history.
 
 - <div lang="jsall">callback</div> := is a function of the form: @function(err, resultPage)@
 - <div lang="swift,objc">callback</div> := called with a "ARTPaginatedResult":#paginated-result<"ARTMessage":#message> object or an error
 
-Further details of the supported @options@ params, see "message history API documentation":/rest/history#channel-history.
```

Comment:
It just repeats the last sentence of the previous paragraph - "...query parameters as specified in the [message history API documentation](). .. Further details of the supported options params, see [message history API documentation]()"